lib/game.ex

```diff
@@ -15,4 +16,29 @@ defmodule TicTacToe.Game do
     Map.replace!(game, :players, Game.get_players(input))
   end
 
+  def add_board(game) do
+    Map.replace!(game, :board, Board.new(3))
+  end
+
+  def make_move(game, move, input \\ CLI) do
+    [ current_player | _tail ] = game.players
+    cond do
+      !Board.space_is_available(game.board.spaces, move) ->
+        input.print "That space isn't available, please choose an open space between 1 and #{length(game.board.spaces)}"
+        new_move = input.get_player_move(current_player.name)
+        make_move(game, new_move, input)
+      Board.space_is_available(game.board.spaces, move) ->
+        new_board = Board.update_at(game.board.spaces, move, current_player.marker)
+        update_in(game, [Access.key(:board), Access.key(:spaces)], &(&1 = new_board))
+    end
+  end
+
+  def player_turn(game) do
+    [ current_player | _tail ] = game.players
+    CLI.display_board(game.board.spaces)
+    move = CLI.get_player_move(current_player.name)
+    new_game = Game.make_move(game, move)
+    CLI.display_board(new_game.board.spaces)
+  end
```

Comment:
Maybe we can try adjusting the flow so it's pipable. 

something like:

```elixir
game
|> CLI.display_board
|> CLI.get_player_move
|> Game.make_move
|> CLI.display_board
```

Reply:
I think I'll actually be able to clean this one up more once I add the main game flow (since a player turn will just consist of display_board and next_move.. maybe just next_move?)
This is what I have now, because the current game only takes one turn:

```Elixir
  def player_turn(game) do
    CLI.display_board(game.board)
    game = Game.next_move(game)
    CLI.display_board(game.board)
  end
```

Would it be redundant to make something like Game.show_board(game) that calls CLI.display_board? It would allow the `game |> Game.show_board |> Game.make_move` flow 🤔